### PR TITLE
Update UsageWithReact.md: `connect` accepts 4 arguments

### PR DESCRIPTION
**Pull request by chicoxyzzy**

### CODE REVIEW

**Comment by omnidan** [comment]
Can you explain the reason for this change? :smiley:

**Comment by chicoxyzzy** [comment]
the reason is `connect` accepts 4 arguments actually

File: docs/basics/UsageWithReact.md [hunk 1]
```diff
@@ -295,7 +295,7 @@ This makes our store instance available to the components below. (Internally, th
...
 
-**Any component wrapped with `connect()` call will receive a [`dispatch`](../api/Store.md#dispatch) function as a prop, and any state it needs from the global state.** The only argument to `connect()` is a function we call a **selector**. This function takes the global Redux store’s state, and returns the props you need for the component. In the simplest case, you can just return the `state` given to you, but you may also wish to transform it first.
+**Any component wrapped with `connect()` call will receive a [`dispatch`](../api/Store.md#dispatch) function as a prop, and any state it needs from the global state.** In most common cases you will pass only first argument to `connect()` which is a function we call a **selector**. This function takes the global Redux store’s state, and returns the props you need for the component. In the simplest case, you can just return the `state` given to you (i.e. pass identity function), but you may also wish to transform it first.
```
Comment (omnidan) [inline docs/basics/UsageWithReact.md]: some small mistakes here (sorry, things like these really bother me :sweat_smile:):

 * ...you will only *pass the* first argument...
 * missing comma after `connect()`

Comment (chicoxyzzy) [inline docs/basics/UsageWithReact.md]: sure! fixed and rebased

**Comment by gaearon** [comment]
Yeah good catch.

**Comment by omnidan** [comment]
Thanks @chicoxyzzy :grin: Seems like @gaearon was faster than me this time (I was about to press the button, damn you!)

**Pull request merged**